src/fu-common.c

```diff
@@ -811,6 +811,12 @@ fu_common_get_path (FuPathKind path_kind)
 		if (tmp != NULL)
 			return g_build_filename (tmp, LOCALSTATEDIR, NULL);
 		return g_build_filename (LOCALSTATEDIR, NULL);
+	/* /boot/efi */
+	case FU_PATH_KIND_ESPBOOTDIR:
+		tmp = g_getenv ("FWUPD_ESPBOOTDIR");
```

Comment:
I would prefer to see this called `FWUPD_ESPDIR` as really the ESP can be mounted anywhere.  We might find it mounted in a non `/boot` location and that will make it clearer.

Reply:
Fixed, thanks.